build

```diff
@@ -47,12 +42,22 @@ while [ $# -gt 0 ]; do
 			resolve_cname=1
 			shift
 			;;
+                --target)
+                        target_dir="$2"
+                        shift 2
+                        ;;
 		*)
 			break
 			;;
 	esac
 done
 
+container_mount_opts=(
+        -v "$PWD/features:/builder/features:ro"
+        -v "$PWD/keyring.gpg:/builder/keyring.gpg:ro"
+        -v "$PWD/$target_dir:/builder/.build"
```

Comment:
probably should use something like `$(realpath "$target_dir")` instead of `$PWD/$target_dir` to handle cases where user provides an absolute path.